### PR TITLE
Added basic installation and usage instructions.

### PR DESCRIPTION
**Pull request by brylie**
Updated with basic installation and usage examples, including section headers. Related issue: #98

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +15,47 @@ For full info, docs and examples, see the
...
 ```
 
-but much more complex strings are possible, and are useful even if your
+In short, you'll get to use the `{{mf}}` helper. 
+
+Extracting translation strings
+------------------
+After you have added several translation strings to your templates, you will need to extract those strings for translation. To do so:
+
+1. make sure your Meteor app is running (by typing `meteor` in the project directory)
```
Comment (DSpeichert) [inline README.md]: I don't think the app needs to be running for mf_extract to work. Did this change recently?

File: README.md [hunk 2]
```diff
@@ -13,13 +15,47 @@ For full info, docs and examples, see the
...
+----
+Once installed, you can start adding the MessageFormat helper to templates:
+
+```
+<p>This string is translatable</p>
+```
+
+...becomes ...
+```
 <p>{{mf 'trans_string' 'This string is translatable'}}</p>
```
Comment (DSpeichert) [inline README.md]: It might be a good idea to mention that developer is responsible for uniqueness of 'trans_string' (the keys).

**Comment by brylie** [comment]
I updated the README based on feedback from @DSpeichert

**Comment by gadicc** [comment]
Hey, that's great!  Thanks guys :)  Don't miss the online docs too which are a lot more comprehensive.  e.g. explains how to install `mf_extract`, but also still uses `mrt` in the quickstart :)  I'll fix that now.

**Pull request merged**